src/vconsole/vconsole-setup.c

```diff
@@ -282,6 +285,7 @@ int main(int argc, char **argv) {
         utf8 = is_locale_utf8();
 
         r = parse_env_file("/etc/vconsole.conf", NEWLINE,
+                           "VC_MAX", &vc_max_s,
```

Comment:
Yea, that was my idea basically. Origianlly I wanted to go for just 6, matching systemd defaults in logind.conf